tests/test_cli_jobs.py

```diff
@@ -51,7 +51,49 @@ def test_cli_jobs_and_stage(mock_request_get, MockResponse):
     mock_request_get.return_value = MockResponse(message)
     result = runner.invoke(jobs, ["test.id", "--stage", "complete"])
     mock_request_get.assert_called_with(
-        "https://api.mapbox.com/tilesets/v1/test.id/jobs?stage=complete&access_token=fake-token"
+        "https://api.mapbox.com/tilesets/v1/test.id/jobs?access_token=fake-token&limit=100&stage=complete"
+    )
+    assert result.exit_code == 0
+    assert json.loads(result.output) == message
+
+
+@pytest.mark.usefixtures("token_environ")
+@mock.patch("requests.Session.get")
+def test_cli_jobs_limit(mock_request_get, MockResponse):
+    """test jobs + stage endpoint"""
+    runner = CliRunner()
+
+    # sends expected request
+    message = {"message": "mock message"}
+    mock_request_get.return_value = MockResponse(message)
+    result = runner.invoke(jobs, ["test.id", "--limit", "10"])
+    mock_request_get.assert_called_with(
+        "https://api.mapbox.com/tilesets/v1/test.id/jobs?access_token=fake-token&limit=10"
+    )
+    assert result.exit_code == 0
+
+
+@pytest.mark.usefixtures("token_environ")
+@mock.patch("requests.Session.get")
+def test_cli_jobs_limit_out_of_range(mock_request_get):
+    runner = CliRunner()
+    result = runner.invoke(jobs, ["test.id", "--limit", "0"])
+    mock_request_get.assert_not_called()
+    assert result.exit_code == 2
+
+
+@pytest.mark.usefixtures("token_environ")
+@mock.patch("requests.Session.get")
+def test_cli_jobs_stage_and_limit(mock_request_get, MockResponse):
+    """test jobs + stage endpoint"""
+    runner = CliRunner()
+
+    # sends expected request
+    message = {"message": "mock message"}
+    mock_request_get.return_value = MockResponse(message)
+    result = runner.invoke(jobs, ["test.id", "--stage", "complete", "--limit", "10"])
+    mock_request_get.assert_called_with(
+        "https://api.mapbox.com/tilesets/v1/test.id/jobs?access_token=fake-token&limit=10&stage=complete"
```

Comment:
`complete` is not a valid stage. `success`?

Reply:
Changed in 49638e11edc0184ec24e4457699cbcd7ea59d995